common/src/main/java/com/wynntils/models/territories/TerritoryModel.java

```diff
@@ -172,6 +176,16 @@ public void onAdvancementUpdate(AdvancementUpdateEvent event) {
         }
     }
 
+    @SubscribeEvent
+    public void onGuildJoined(GuildEvent.Joined e) {
+        scheduleTerritoryUpdate();
```

Comment:
I see this is already merged, and maybe this comment does not matter, but if you join a guild, it will take 15 s to update the territory. I was thinking that this should trigger an instant update, and then the next after 15 s.

Reply:
It actually does work like that, since we put 0 as the initialDelay in scheduleWithFixedDelay so it's immediately called

Reply:
Ah, I see. That's great. Thanks!